src/App.tsx

```diff
@@ -14,12 +15,9 @@ class App extends React.Component {
 				<Router>
 					<Navigation />
 					<Switch>
-						<Route path="/results">
-							<Results />
-						</Route>
-						<Route path="/">
-							<Home />
-						</Route>
+						<Route path="/results/:id" component={FeedPage} />
+						<Route path="/results" component={Results} />
+						<Route path="/" component={Home} />
```

Comment:
Order here matters because if `/` goes first, then when you go to `/results` the Home page is rendered. Similarly, if `/results` appears before `/results/:id`, then when you're on a feed page, the Results page is rendered. So the new ordering goes from most specific to least specific.